src/Test/StateMachine/Internal/Sequential.hs

```diff
@@ -66,28 +69,37 @@ import           Test.StateMachine.Types
 
 ------------------------------------------------------------------------
 
+-- | Lift a generator of untyped commands with reference placeholders
+--   into a generator of lists of untyped internal commands.
```

Comment:
Kept calling them `untyped`, but I'm happy to change it. My reasoning was that `resp` is hidden...